action.yml

```diff
@@ -84,30 +77,37 @@ runs:
         pip3 -v install distgen
         DG=$HOME/.local/bin/dg make generate-all
 
-    - name: Check if .exclude-${{ inputs.tag }} is present in version directory
+    - name: Check if .exclude-${{ inputs.tag }} is present in ${{ inputs.dockerfile_path }} directory
       id: check_exclude_file
       # https://github.com/marketplace/actions/file-existence
       uses: andstor/file-existence-action@v1
       with:
-        files: "${{ inputs.version }}/.exclude-${{ inputs.tag }}"
+        files: "${{ inputs.dockerfile_path }}/.exclude-${{ inputs.tag }}"
 
-    - name: Check if ${{ inputs.dockerfile }} is present in version directory
+    - name: Check if ${{ inputs.dockerfile }} is present in ${{ inputs.dockerfile_path }} directory
       if: steps.check_exclude_file.outputs.files_exists == 'false'
       id: check_dockerfile_file
       # https://github.com/marketplace/actions/file-existence
       uses: andstor/file-existence-action@v1
       with:
-        files: "${{ inputs.version }}/${{ inputs.dockerfile }}"
+        files: "${{ inputs.dockerfile_path }}/${{ inputs.dockerfile }}"
         allow_failure: true # fails the Action if Dockerfile is missing
 
+    - name: Get cleaned path to Dockerfile
+      id: clean_dockerfile_path
+      shell: bash
+      run: |
+        tmp="${{ inputs.dockerfile_path }}"
+        echo "::set-output name=clean_dockerfile_path::${tmp//./}"
```

Comment:
Please do not run `id` the same as `output`.
Like `steps.clean_dockerfile_path.outputs.clean_dockerfile_path`.
It would be nice to have something like:

`steps.clean_path.outputs.clean_dockerfile_path`